cypress/integration/eholdings/provider-search.spec.js

```diff
@@ -10,48 +12,37 @@ describe('ui-eholdings: Search providers', () => {
     });
 
     it('should display two results', () => {
-      cy.get('#search-results-content').find('li').should(li => expect(li).to.have.length(2));
+      cy.expect(List().has({ count: 2 }));
     });
   });
 
   describe('searching by single tag', () => {
     beforeEach(() => {
-      cy.get('#accordion-toggle-button-accordionTagFilter').click();
-      cy.get('#accordionTagFilter input[type="checkbox"]').click();
-
-      cy.get('#selectTagFilter-input').type('important');
-      cy.get('#accordionTagFilter li[class*="multiSelectOption--"]:first-child').click();
+      cy.do([
+        Button('Tags').click(),
+        Checkbox('Search by tags only').click(),
+        MultiSelect().select(['important']),
+      ]);
     });
 
+    // FIXME: There are no results
     it('should display list of providers with important tag', () => {
-      cy.get('#search-results-content').find('li').should(li => expect(li).to.have.length(2));
-    });
-
-    afterEach(() => {
-      cy.get('#accordionTagFilter button[icon="times-circle-solid"').click();
-      cy.reload();
+      cy.expect(List().has({ count: 2 }));
     });
   });
 
   describe('searching by multiple tags', () => {
     beforeEach(() => {
-      cy.get('#accordion-toggle-button-accordionTagFilter').click();
-      cy.get('#accordionTagFilter input[type="checkbox"]').click();
-
-      cy.get('#selectTagFilter-input').type('important');
-      cy.get('#accordionTagFilter li[class*="multiSelectOption--"]:first-child').click();
-
-      cy.get('#selectTagFilter-input').type('urgent');
-      cy.get('#accordionTagFilter li[class*="multiSelectOption--"]:first-child').click();
+      cy.do([
+        Button('Tags').click(),
+        Checkbox('Search by tags only').click(),
+        MultiSelect().select(['important', 'urgent']),
+      ]);
     });
 
+    // FIXME: There are no results
```

Comment:
Is this test is failing now or what? Like I mentioned above please change the message.

Reply:
It's failing because the test expects at least one MARC record with `Robert C. Klove papers.` query, but API returns nothing

Reply:
That sounds like it would be the perfect text for the `FIXME`